content/en/docs/concepts/security/index.md

```diff
@@ -569,366 +556,25 @@ An authorization policy includes a selector, an action, and a list of rules:
-{{< /text >}}
-
-The following example shows an authorization policy that denies requests if the
-source is not the `foo` namespace:
-
-{{< text yaml >}}
-apiVersion: security.istio.io/v1beta1
-kind: AuthorizationPolicy
-metadata:
- name: httpbin-deny
- namespace: foo
-spec:
- selector:
-   matchLabels:
-     app: httpbin
-     version: v1
- action: DENY
- rules:
- - from:
-   - source:
-       notNamespaces: ["foo"]
-{{< /text >}}
-
-The deny policy takes precedence over the allow policy. Requests matching allow
-policies can be denied if they match a deny policy. Istio evaluates deny
-policies first to ensure that an allow policy can't bypass a deny policy.
-
-#### Policy Target
-
-You can specify a policy's scope or target with the
+Policy Scope can be specified with the
 `metadata/namespace` field and an optional `selector` field.
-A policy applies to the namespace in the `metadata/namespace` field. If
-set its value to the root namespace, the policy applies to all namespaces in a
-mesh. The value of the root namespace is configurable, and the default is
-`istio-system`. If set to any other namespace, the policy only applies to the
-specified namespace.
-
-You can use a `selector` field to further restrict policies to apply to specific
-workloads. The `selector` uses labels to select the target workload. The
-selector contains a list of `{key: value}` pairs, where the `key` is the name of
-the label. If not set, the authorization policy applies to all workloads in the
+If policy target is set to the root namespace (`istio-system` by default), the policy applies to all namespaces in the
+mesh. A `selector` field can further restrict policies to apply to specific
+workloads. The `selector` uses labels (a list of `{key: value}` pairs) to select the target workload. If not set, the authorization policy applies to all workloads in the
 same namespace as the authorization policy.
 
-For example, the `allow-read` policy allows `"GET"` and `"HEAD"` access to the
-workload with the `app: products` label in the `default` namespace.
-
-{{< text yaml >}}
-apiVersion: security.istio.io/v1beta1
-kind: AuthorizationPolicy
-metadata:
-  name: allow-read
-  namespace: default
-spec:
-  selector:
-    matchLabels:
-      app: products
-  action: ALLOW
-  rules:
-  - to:
-    - operation:
-         methods: ["GET", "HEAD"]
-{{< /text >}}
+Value matching in authorization policies supports Exact match, Prefix match with an ending `"*"`,
+Suffix match with a starting `"*"`, Presence match with `fieldname: ["*"]` format to specify
+that a field must be present.
+At the same time, user can use Exclusion matching to match negative conditions like `notValues` in the `when` field, `notIpBlocks`
+in the `source` field, `notPorts` in the `to` field. Check [Security Tasks]() for further details.
 
-#### Value matching
+In the [Security Policy Examples](docs/ops/configuration/security/security-policy-examples/), we introduce more policy use cases like:
```

Comment:
I think you should make sure the new content is ready when deleting it from the concept page. You could do it in a single PR that adds the new content in example/task/reference page while deleting it from concept page.

Otherwise there is chance that some contents are removed in this PR but forget to be added in other places, and more importantly, separating the change will make it harder to review and compare the newly added content to the old one, and it puts the preminary.istio.io in an incomplete status until all changes are merged.

Reply:
Thanks for your suggestion. Yes, the changes to the authz part will be in the same PR. But let's first have consensus of how will the concept page look like. The concept page changes will be finished in the 1st commit, and I will commit the tasks/reference changes in this PR as a different commit.

Reply:
Once we reach agreement on the concept page, @justinpettit will work with me on finding a section that fits for the delete content. In this way, we can greatly reduce moving the content around. Thanks~